src/main/java/com/onarandombox/MultiverseCore/commands/MultiverseCommand.java

```diff
@@ -15,9 +17,20 @@ public class MultiverseCommand extends BaseCommand {
     protected final MVWorldManager worldManager;
     protected final CommandFlagsManager flagsManager;
 
+    private String flagGroupName;
+
     protected MultiverseCommand(@NotNull MultiverseCore plugin) {
         this.plugin = plugin;
         this.worldManager = plugin.getMVWorldManager();
         this.flagsManager = plugin.getCommandManager().getFlagsManager();
     }
+
+    protected void registerFlagGroup(@NotNull CommandFlagGroup flagGroup) {
+        flagsManager.registerFlagGroup(flagGroup);
```

Comment:
Awesome. Only other thing I think is maybe preventing this from being called again.

Reply:
Make sense, shoulda thought of that